tf2onnx/optimizer/optimizer_base.py

```diff
@@ -13,10 +14,13 @@ class GraphOptimizerBase(object):
     def __init__(self, name, debug=False):
         self._debug = debug
         self._name = name
+        self._log = logging.getLogger("tf2onnx.optimizer.%s" % self._name)
 
     def optimize(self, graph):
         original_node_statistics = graph.dump_node_statistics()
         graph = self._optimize(graph)
+        graph.delete_unused_nodes(graph.outputs)
+        graph.topological_sort(graph.get_nodes())
```

Comment:
is topo-sort a common requirement for all optimizers?
i suggest let optimizer itself to call topo sort if topo order is required for that specific optimizer.

Reply:
not necessary, I will remove it.

my initial thought is to make sure every optimizer ends with a legal onnx graph.